src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/DatesController.kt

```diff
@@ -99,6 +99,6 @@ class DatesController(
     ],
   )
   fun currentHardStopCutoffDate(): HardStopCutoffDate {
-    return HardStopCutoffDate(releaseDateService.getCutOffDateForLicenceTimeOut(LocalDate.now()))
```

Comment:
we have a clock internal to the service so using this but support overriding for tests